apps/api-gateway/schema.graphql

```diff
@@ -0,0 +1,1451 @@
+directive @enum(subgraph: String, value: String) on ENUM_VALUE
```

Comment:
The schema.graphql file is added but was previously in .gitignore. This could break builds if the file is expected to be generated rather than committed. Verify if this file should be generated during build or committed to version control.